subsys/bluetooth/host/conn.c

```diff
@@ -302,6 +302,10 @@ static void tx_notify(struct bt_conn *conn)
 }
 #endif	/* CONFIG_BT_CONN_TX */
 
+#if defined(CONFIG_BT_CONN)
+static void auto_initiated_procedures(struct k_work *work);
+#endif	/* defined(CONFIG_BT_CONN) */
```

Comment:
```suggestion
#endif /* defined(CONFIG_BT_CONN) */
```

Space instead of tab? :)